l10n_nl_hr_employee_name/tests/test_l10n_nl_hr_employee_name.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp.tests.common import TransactionCase
+
+
+class TestL10nNlPartnerName(TransactionCase):
```

Comment:
from the looks of it, you should be able to import the test case classes from https://github.com/OCA/partner-contact/blob/8.0/partner_firstname/tests, override some strategic functions where you create employee records instead of partners, and get all the test code for free thanks to late binding (only works if you follow my advice above to recreate the `protocol` of partner_firstname by providing functions with the same name.

remember that unittest autodetects tests classes by class name, so you must alias the classes you import or just import the module containing it.